docs/src/Module.doc.js

```diff
@@ -36,6 +36,22 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'expandedIndex',
+        type: '?number',
+        required: false,
+        description: [
+          'The index of the item that the expand/collapse state can be controlled programatically from an external component',
+        ],
+      },
+      {
+        name: 'onExpandedChange',
+        type: '(?number) => void',
+        required: false,
+        description: [
+          'The callback function that controls the expand/collapse state of an item if controlled programatically from an external component',
```

Comment:
Can we add

The callback receives the index of the module component when expanded, and null when collapsed.